src/context/index.ts

```diff
@@ -40,6 +42,26 @@ export const setupContext = async (config: Config): Promise<DirectoryContext | Y
     );
   }
 
+  ((config: Config) => {
+    if (config.AUTH0_INCLUDED_ONLY === undefined) return;
+
+    if (config.AUTH0_INCLUDED_ONLY.length === 0) {
+      throw new Error(
+        `Need to define at least one resource type in AUTH0_INCLUDED_ONLY configuration.`
```

Comment:
Wdyt about including a link to the docs? e.g. "For more info: https://github.com/auth0/auth0-deploy-cli/blob/master/docs/configuring-the-deploy-cli.md"

Reply:
Makes total sense, will add!